src/rebar3_hex.erl

```diff
@@ -25,9 +25,7 @@ init(State) ->
                                                    rebar3_hex_key,
                                                    rebar3_hex_owner,
                                                    rebar3_hex_repo,
-                                                   rebar3_hex_docs,
                                                    rebar3_hex_search,
-                                                   rebar3_hex_revert,
```

Comment:
revert is gone, but unlike docs this won't be re-introduced, it's just a property of publish now.